recipes/libpyvinyl/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set version = "1.3.0" %}
+
+package:
+  name: libpyvinyl
+  version: {{ version }}
+
+source:
+  url: https://github.com/PaNOSC-ViNYL/libpyvinyl/archive/v{{ version }}.tar.gz
+  sha256: 1e83f89b3b702d6b527bba6a2efd64b4c76cad23ad8a8c305ad356381bcd4e93
+
+  #url: https://pypi.io/packages/source/l/libpyvinyl/libpyvinyl-{{ version }}.tar.gz
+  ##NB: get sha256 from https://pypi.org/project/libpyvinyl/#files
+  #sha256: 5031eacda47519734adf0cea21c684bb368659f08f8df48a698b56fb14c837e3
```

Comment:
Why not the PyPI tarball? Also, can you remove these commented out lines?

Reply:
Because of an issue in the upstream project (we did try the pypi source first), meaning that the pypi source dists are broken. This has been reported upstream, but in the meantime the github tarballs are needed as a workaround.